src/main/java/io/minio/MinioClient.java

```diff
@@ -2303,13 +2303,9 @@ private ListMultipartUploadsResult listIncompleteUploads(String bucketName, Stri
     throws InvalidBucketNameException, NoSuchAlgorithmException, InsufficientDataException, IOException,
            InvalidKeyException, NoResponseException, XmlPullParserException, ErrorResponseException,
            InternalException {
-    if (maxUploads < 0 || maxUploads > 1000) {
-      maxUploads = 1000;
-    }
-
     Map<String,String> queryParamMap = new HashMap<>();
     queryParamMap.put("uploads", "");
-    queryParamMap.put("max-uploads", Integer.toString(maxUploads));
+    queryParamMap.put("max-uploads", Integer.toString(maxUploads < 0 || maxUploads > 1000 ? 1000 : maxUploads));
```

Comment:
It would be better to declare `maxUploadsString` than using ternary operator.  Its better to avoid ternary operator.